frontend/src/scenes/events/eventsTableLogic.js

```diff
@@ -161,6 +168,16 @@ export const eventsTableLogic = kea({
             (events, newEvents) => formatEvents(events, newEvents, props.apiUrl),
         ],
         columnConfig: [() => [userLogic.selectors.user], (user) => user?.events_column_config?.active || 'DEFAULT'],
+        exportUrl: [
+            () => [selectors.eventFilter, selectors.orderBy, selectors.properties],
+            (eventFilter, orderBy, properties) =>
+                `/api/event.csv?${toParams({
+                    properties,
+                    ...(props.fixedFilters || {}),
+                    ...(eventFilter ? { event: eventFilter } : {}),
+                    orderBy: [orderBy],
+                })}`,
+        ],
```

Comment:
Warning: incoming some very kea-specific context :).

When using `props` inside a selector, we should also put them in the input selector array, otherwise our new selector won't be recalculated when the prop's value changes. Selectors are just functions `(state, props) => state.something.whatever`. In this case you'd use `(_, props) => props.fixedFilters` as the 4th selector in the array.

However... `props.fixedFilters` probably changes very often, as it's a dynamically created object [here](https://github.com/PostHog/posthog/blob/3f6d9799f8f9ef9b182342480daef6aef68eb4d8/frontend/src/scenes/persons/Person.tsx#L76) and elsewhere. Selectors check for reference equality, and every time that component renders, a new object is created, and it won't referentially equal the old props.

However... this entire logic has a `key` that's built with `JSON.stringify(props.fixedFilters)`, meaning that if the fixed filters change in any way in their content, it'll be a new logic anyway...

Thus... the code you have is correct, but I'd like to somehow pass on this context on the `...(props.fixedFilters || {})` line. Perhaps just with a comment `// this never changes`... Otherwise the immediate wish is to refactor it to recalculate when changed.